addon-test-support/helpers.js

```diff
@@ -7,6 +7,9 @@ const DEFAULT_EVENT_OPTIONS = { canBubble: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
 
+import config from 'dummy/config/environment';
```

Comment:
Right, I don't think we can do this (at least not reliably). We will need to think of another way...